src/components/fingerprint/fingerprint.html

```diff
@@ -0,0 +1,24 @@
+<transition name="modal">
+<div class="modal-mask" @click="close">
+    <div class="modal-container" @click.stop style="height:100%;width:65%;overflow-y:auto">
+        <span @click="close" class="close">&times;</span>
+        <center>
+            <h2 >Verifying friend: {{ friendname }}</h2>
+        </center>
+        <center style="font-size:2em">
+          <img v-if="stream == null" v-bind:src="QRCodeURL" width="512" height="512"></img>
+	  <video v-if="stream != null" id="video" width="512" height="512"></video>
+	  <div>
+	    <button class="btn btn-success" @click="scanQRCode()">Scan your friend's QR code</button>
+	    <br/>
+	    <h4>Or you can read these numbers out to each other (they should match)</h4>
```

Comment:
This is a little unclear. Maybe change this to read "If you're unable to scan the QR code, you can still verify your friend's identity using the sequence of numbers below. Read the sequence to each other and ensure that they match."

Reply:
We might want to add a TODO to add a how-to article that describes a safe way to transmit the numbers to each other.